benchmark_runner/common/oc/oc.py

```diff
@@ -1636,6 +1643,14 @@ def generate_cnv_must_gather(self, destination_path: str = '/tmp', cnv_version:
                     raise RuntimeError(f"Failed to remove folder {folder_path}: {remove_error}")
             raise RuntimeError(f"Failed to generate CNV must-gather logs for version {cnv_version}: {e}")
 
+    def generate_must_gather(self, run_artifacts_path, cnv_version, odf_version):
+        """
+        This method generates must gather and saves it in the run artifacts path
+        """
+        self.generate_ocp_must_gather(destination_path=run_artifacts_path)
+        self.generate_cnv_must_gather(destination_path=run_artifacts_path, cnv_version=cnv_version)
+        self.generate_odf_must_gather(destination_path=run_artifacts_path, odf_version=odf_version)
```

Comment:
Why do you need the versions here?

I'm approving it because the logic hasn't changed from before, just wonder why you need to do this.